components/menu-dropdown/button-trigger.jsx

```diff
@@ -19,6 +19,8 @@ import Button from '../button';
 
 // ### Children
 import { MENU_DROPDOWN_TRIGGER } from '../../utilities/constants';
+import checkProps from './check-props';
```

Comment:
button-trigger should have it's own check props file. Otherwise the same check props will be run on dropdown and button trigger components.